apps/build.sh

```diff
@@ -196,26 +196,24 @@ for x in $IMAGES ; do
 			run docker push ${ct_base}:$TAG-$ARCH
 		fi
 
+		run docker manifest create ${ct_base}:${H_BUILD}_$TAG ${ct_base}:$TAG-$ARCH
+		run docker manifest create ${ct_base}:${LATEST} ${ct_base}:$TAG-$ARCH
+
 		var="EXTRA_TAGS_$ARCH"
 		for t in $(eval echo "\$$var") ; do
```

Comment:
It was just a lot easier to do it here since this logic of sourcing docker-build.conf correctly is done here. Otherwise we have to copy that code there and most of publish is done as python.